pkg/actors/actors.go

```diff
@@ -239,8 +235,20 @@ func newActorsWithClock(opts ActorsOpts, clock clock.WithTicker) (ActorRuntime,
 
 	a.timers.SetExecuteTimerFn(a.executeTimer)
 
-	if opts.Config.SchedulerClients != nil {
+	if opts.Config.SchedulerClients != nil && opts.SchedulerReminders {
```

Comment:
We can simplify this. If the preview feature is on but there is no client to scheduler, than it should error out and not init.

Reply:
done